Kickstarter-iOS/AppDelegate.swift

```diff
@@ -416,9 +416,16 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
         return
       }
 
-      print("🔴 Remote Config SDK Activation Failed with Error: \(remoteConfigActivationError.localizedDescription)")
+      let errorAsNSError = remoteConfigActivationError as NSError
 
-      Crashlytics.crashlytics().record(error: remoteConfigActivationError)
+      if errorAsNSError.domain == RemoteConfigErrorDomain,
+        errorAsNSError.code == RemoteConfigError.internalError.rawValue {
```

Comment:
Dug a bit into the Firebase code - this is the correct domain and error for what they're sending us (error `8003`). The naming is a bit confusing, so it's possible we'll accidentally catch an error we may have been interested in.

Reply:
cc @ifosli since you hated this error!